src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        return String.valueOf(
+                Colors.values()[new Random().nextInt(
+                        Colors.values().length
+                )]
+        );
```

Comment:
```suggestion
        return Colors.values()[random.nextInt(Colors.values().length)].name();
```
create a global Random field and use it here